contrib/firmware_packager/firmware_packager.py

```diff
@@ -38,23 +39,41 @@ def cd(path):
   <project_license>proprietary</project_license>
   <developer_name>{developer_name}</developer_name>
   <releases>
-    <release version="{release_version}" date="{date}">
+    <release urgency="{release_urgency}" version="{release_version}" date="{date}">
+      <checksum filename="firmware.bin" target="content"/>
       <description>
         <p>{release_description}</p>
+        <p>{release_features}</p>
       </description>
     </release>
   </releases>
   <custom>
     <value key="LVFS::VersionFormat">{version_format}</value>
     <value key="LVFS::UpdateProtocol">{update_protocol}</value>
+    <value key="LVFS::RequireTest">true</value>
   </custom>
+  <categories>
+    <category>{firmware_category}</category>
+  </categories>
```

Comment:
i think do a for loop here, splitting the category on "," or something. Quite a few devices have multiple categories.

Reply:
Splitting them with |. Thanks